mantidproject/mantid
# Update and move ORNL SANS reduction to Mantid repo

The ORNL SANS reduction docs date from 2010 and were part of the old wiki. Transfer then to rst and put them in the proper place in the Mantid repo.

To test:
- Read the docs, under concepts/ORNL_SANS_Reduction
- Start the HFIR UI and see that the link on the first tab is updated and links to the right place.
- Start the EQSANS UI and see that nothing is broken.
